content/cloud-docs/run/ui.mdx

```diff
@@ -63,6 +63,14 @@ If you would rather automatically apply plans that don't have errors, you can [e
 
 To help you review proposed changes, Terraform Cloud can automatically run [speculative plans][speculative plan] for pull requests or merge requests.
 
+## Upgrading Terraform using Speculative Plans
+
+Using the UI, you can start a new speculative plan using the current configuration version but any terraform version available to the organization. After clicking **Start New Run**, use the select list to choose **Plan only** as the run type. This run mode reveals a new option to choose the specific terraform version.
+
+![Screenshot: Clicking on the "Start New Run" button to try a new terraform version](/img/docs/run-plan-only-upgrade-ui.png)
```

Comment:
My recommendation is to remove this screenshot. The reason why is that screenshots are difficult to maintain as our UI evolves, and I'm not sure they're necessary for contained dialog boxes like this. Can we try without?